src/Parser.elm

```diff
@@ -1,6 +1,7 @@
 module Parser exposing (..)
```

Comment:
Now that we have chevrotain, we don't need our old Parser anymore. Maybe we should leave it for now for reference, but ultimately it should be deleted.